applications/serial_lte_modem/doc/NRFCLOUD_AT_commands.rst

```diff
@@ -37,7 +37,7 @@ The ``<op>`` parameter accepts the following integer values:
 
 * ``0`` - Disconnect from the nRF Cloud service.
 * ``1`` - Connect to the nRF Cloud service.
-* ``2`` - Send a message in the JSON format to the nRF Cloud service.
+* ``2`` - Send an arbitrary message to the nRF Cloud service in ``slm_data_mode``.
```

Comment:
Replacing this removes possibility of sending messages in JSON format. Is the JSON format not needed any more?

Reply:
> Replacing this removes possibility of sending messages in JSON format. Is the JSON format not needed any more?

nRF Cloud used to accept any JSON message, but now requires "appID" so we change to format JSON message in SLM for arbitray data sending.